feature/home/src/main/java/com/tapptitude/home/presentation/HomeScreen.kt

```diff
@@ -0,0 +1,114 @@
+package com.tapptitude.home.presentation
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import coil.compose.AsyncImage
+import com.tapptitude.config.BuildConfig
+import com.tapptitude.core.model.Image
+import com.tapptitude.feature.home.R
+import com.tapptitude.session.model.LoggedIn
+import com.tapptitude.session.model.LoggedOut
+import com.tapptitude.session.model.LoginState
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+internal fun HomeRoute(
+    modifier: Modifier = Modifier,
+    viewModel: HomeViewModel = koinViewModel()
```

Comment:
I suggest we make another branch using hilt for compose
Then we can either keep it up to date separately or replace the koin impl altogether

Reply:
I think you can create an issue for this in order to track it.